annotation-client/src/test/java/eu/europeana/annotation/client/integration/webanno/BaseWebAnnotationTest.java

```diff
@@ -50,7 +50,7 @@
 	public static final String WHITELIST_ENTRY = "/whitelist/entry.json";
 	public static final String SUBTITLE_MINIMAL = "/subtitle/minimal.json";
 	public static final String CAPTION_MINIMAL = "/caption/minimal.json";
-	public static final String LINK_FOR_CONTRIBUTING = "/linkforcontributing/link_for_contributing.json";
+	public static final String LINK_FOR_CONTRIBUTING = "/linkforcontributing/link_for_contributing_body_string.json";
```

Comment:
add example with the 2 formats to the templates file: it is the [jsonld.jsp](https://github.com/europeana/annotation/blob/develop/annotation-web/src/main/webapp/jsp/template/jsonld.jsp) file which is linked in the description of the create method

Reply:
Done.